installation/quest/download.md

```diff
@@ -1,3 +1,14 @@
 # Downloading Noodle Extensions
 
-[TBD] cal117 is working on this page.
+You can download Noodle Extensions for quest from the following sources:
+
+- [noodleextensions.com](https://noodleextensions.com/#download)
+- [Beat Saber Modding Group](https://discord.gg/beatsabermods)
+- [GitHub](https://github.com/StackDoubleFlow/NoodleExtensions/releases/latest)
+
+Make sure to download the file that has the `.qmod` file extension - `.so` and `.zip` are invalid.
```

Comment:
Maybe add "or any other file types are invalid" at the end because test review :) 💯

Reply:
Okay :)

Reply:
```suggestion
Make sure to download the file that has the `.qmod` file extension - any other file types are invalid.
```

Reply:
Commited